tests/s3_mock_server_tests.c

```diff
@@ -144,3 +144,45 @@ TEST_CASE(async_access_denied_from_complete_multipart_mock_server) {
 
     return AWS_OP_SUCCESS;
 }
+
+TEST_CASE(get_object_modified_mock_server) {
+    (void)ctx;
+
+    struct aws_s3_tester tester;
+    ASSERT_SUCCESS(aws_s3_tester_init(allocator, &tester));
+    struct aws_s3_tester_client_options client_options = {
+        .part_size = 64 * 1024,
+        .tls_usage = AWS_S3_TLS_DISABLED,
+    };
+
+    struct aws_s3_client *client = NULL;
+    ASSERT_SUCCESS(aws_s3_tester_client_new(&tester, &client_options, &client));
+
+    /* Check the README part for getobject mismatch */
+    struct aws_byte_cursor object_path = aws_byte_cursor_from_c_str("/get_object_modified");
```

Comment:
what README part are you talking about?

Reply:
The mock server README. Updated the README and the comment